certified-connectors/VirusTotal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3325 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.4",
```

Comment:
Just wanted to double check.. did you run swagger validator on this to make sure there are no issues?

Reply:
It pass all the validations and certifications so I guess it does